<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emeterForm</class>
 <widget class="QWidget" name="emeterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Frequency">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mains Voltage- Y</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_YVoltage">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Frequency</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_RVoltage">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>KWH</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mains Voltage- B</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_BVoltage">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mains Voltage- R</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_KWH">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="emeterbackButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>751</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enengy Meter Readings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Read Emeter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
